docs/virtual_machines/disks_and_volumes.md

```diff
@@ -654,12 +720,19 @@ to a VM.
     status: {}
 
 ### configMap
-
 A `configMap` is a reference to a
 [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/)
-in Kubernetes. An extra `iso` disk will be allocated which has to be
+in Kubernetes. 
+A `configMap` can be presented to the VM as disks or as a filesystem. Each method is described in the following
+sections and both have some advantages and disadvantages, i.e. `disks` does not support dynamic change propagation and
```

Comment:
```suggestion
sections and both have some advantages and disadvantages, i.e. `disk` does not support dynamic change propagation and
```

Reply:
Addressed this and the rest of occurrences.